developerscircleeldoret.apib

```diff
@@ -0,0 +1,41 @@
+FORMAT: 1A
+HOST: http://polls.apiblueprint.org/
+
+# Developers Circle Eldoret
+
+Connecting professionals, hobbyist, hippies, anyone, we don't care. All we need is a heart for code.
```

Comment:
I like this tag line 😂

Reply:
Could you provide further documentation on the readme.md on how to access and use this documentation?

Reply:
@miami78, I appreciate the suggestion but this was just a setup so that we can have all the people on the same page. We actually don't have an API, I haven't worked on the backend at all and I see you guys haven't too, there is still so much we need to clear up on that end. I had this up so that I can guide you guys on how to set up a live preview, for those not familiar with Apiary, on your editors (If you collaboratively choose to go the Apiary way).

Reply:
@yegow ,oooh, now i get it, awesome, let me write a contribution.md so we can pick up from there

Reply:
Sounds awesome, @miami78. Heads up when you're done.

Reply:
> @yegow ,oooh, now i get it, awesome, let me write a contribution.md so we can pick up from there

As this is being addressed in #4 , I think it is safe to merge this @miami78 .

Reply:
@yegow its done @MarvinKweyu,I've seen it and i approve